lib/liquid/tag.rb

```diff
@@ -13,7 +13,15 @@ def parse(tag_name, markup, tokenizer, parse_context)
         tag
       end
 
+      def disable_nested_tags(*tags)
```

Comment:
I would go with `disable_tags` since we don't really use the "nested" elsewhere in the implementation.

Reply:
Fair